snmp/datadog_checks/snmp/data/default_profiles/bluecat-server.yaml

```
@@ -0,0 +1,79 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-host-resources.yaml
+  - _generic-ucd.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.13315.2.*
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "bluecat"
+
+metrics:
+  - MIB: BCN-DHCPV4-MIB
+    table:
+      OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2
+      name: bcnDhcpv4SubnetTable
+    symbols:
+      - OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.4
+        name: bcnDhcpv4SubnetFreeAddresses
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.1
+          name: bcnDhcpv4SubnetIP
+        tag: bcn_dhcpv4_subnet_ip
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.2
+          name: bcnDhcpv4SubnetMask
+        tag: bcn_dhcpv4_subnet_mask
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.3
+          name: bcnDhcpv4SubnetSize
+        tag: bcn_dhcpv4_subnet_size
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.5
+          name: bcnDhcpv4SubnetLowThreshold
+        tag: bcn_dhcpv4_subnet_low_threshold
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.2.1.6
+          name: bcnDhcpv4SubnetHighThreshold
+        tag: bcn_dhcpv4_subnet_high_threshold
+  - MIB: BCN-DHCPV4-MIB
+    table:
+      OID: 1.3.6.1.4.1.13315.3.1.1.2.2.3
+      name: bcnDhcpv4PoolTable
+    symbols:
+      - OID: 1.3.6.1.4.1.13315.3.1.1.2.2.3.1.5
+        name: bcnDhcpv4PoolFreeAddresses
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.3.1
```

http://www.circitor.fr/Mibs/Html/B/BCN-DHCPV4-MIB.php#bcnDhcpv4PoolStartIP
```suggestion
          OID: 1.3.6.1.4.1.13315.3.1.1.2.2.3.1.1
```

---

Good catch!